client/src/components/TreeDropdownField/TreeDropdownField.js

```diff
@@ -116,7 +114,7 @@ class TreeDropdownField extends Component {
    * @return {Object}
    */
   getVisibleTree() {
-    return this.findTreeByPath(this.props.tree, this.props.visible);
+    return this.props.findTreeByPath(this.props.tree, this.props.visible);
```

Comment:
I'm not sure why this is both a prop and an import. Surely it should just be `findTreeByPath` not `this. findTreeByPath` nor `this.props. findTreeByPath`?